components/texts/text-list.css

```diff
@@ -0,0 +1,59 @@
+.texts {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.heading, .tx, .gr-tx {
+    margin: 0.5rem;
+}
+.medium-text {
+    font-size: 20px;
+}
+.gr-tx {
+    color: darkgray;
+}
+
+.center-text {
+    width: 500px;
+    margin: auto;
+}
+
+/* LISTS */
+.spaced-list, .stacked-list {
+    margin-top: 0.5rem;
+    margin-bottom: 1.5rem;
+}
+.sl-ul li {
+    font-size: 18px;
+    margin: 0.5rem;
+}
+
+.notification-card {
+    display: flex;
+    padding: 0rem 4rem;
+    list-style: none;
+    justify-content: center;
+    align-items: center;
+    padding: 1rem 0rem 1rem 0rem;
```

Comment:
use padding: 16px 0px;  Other two will automatically take from this two..

Reply:
padding: 1rem 0rem ;
means 1rem for top and bottom and 0rem for both right and left. It is one of the shorthand formats

.

Reply:
Yes Thats what i wanted to say exactly